python/cudf/cudf/core/frame.py

```diff
@@ -2312,29 +2311,31 @@ def drop_duplicates(
 
     def replace(self, to_replace, replacement):
```

Comment:
Type annotations?  It's OK to use `Any` if `to_replace` and `replacement` can be arbitrary objects. But the return type is probably more concrete.

Reply:
Introduced the type annotations here